src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyAlone.java

```diff
@@ -74,4 +74,7 @@ void method6(int a) {
         }{}; // violation
         };
     }
+
+    public @interface MyAnnotation {
+        int someValue();} //violation
```

Comment:
Please copy similar new cases from `InputRightCurlyOtherAlone.java` to here.
And please copy this current case to the other options. I don't see it in them.

Reply:
@rnveach I have added the case across all options. Like I mentioned [here](https://github.com/checkstyle/checkstyle/pull/7039#discussion_r332463068) the violating case for `RightCurlyOption.SAME` is present in `InputRightCurlyLeft.java`. And as this case is a violation for `RightCurlyOption.SAME` it is not present in `InputRightCurlySame.java`.